audio/beets/Portfile

```diff
@@ -0,0 +1,601 @@
+                    size    1694845
+
+python.default_version 39
+
+depends_build-append \
+                    port:py${python.version}-setuptools
+
+if {${name} eq ${subport}} {
+    depends_lib-append \
+                    port:py${python.version}-Pillow \
+                    port:py${python.version}-acoustid \
+                    port:py${python.version}-beautifulsoup4 \
+                    port:py${python.version}-confuse \
+                    port:py${python.version}-discogs-client \
+                    port:py${python.version}-flask \
+                    port:py${python.version}-flask-cors \
+                    port:py${python.version}-gmusicapi \
+                    port:py${python.version}-jellyfish \
+                    port:py${python.version}-langdetect \
+                    port:py${python.version}-last \
+                    port:py${python.version}-mediafile \
+                    port:py${python.version}-mpd2 \
+                    port:py${python.version}-munkres \
+                    port:py${python.version}-musicbrainzngs \
+                    port:py${python.version}-mutagen \
+                    port:py${python.version}-rarfile \
+                    port:py${python.version}-reflink \
+                    port:py${python.version}-requests \
+                    port:py${python.version}-requests-oauthlib \
+                    port:py${python.version}-six \
+                    port:py${python.version}-soco \
+                    port:py${python.version}-unidecode \
+                    port:py${python.version}-xdg \
+                    port:py${python.version}-yaml
+
+    depends_run-append \
+                    port:chromaprint \
+                    port:ffmpeg \
+                    port:flac \
+                    port:mp3val \
+                    port:ImageMagick \
+                    port:ipfs
+
+    post-destroot {
+        xinstall -d ${destroot}${prefix}/share/man/man1
+        xinstall -m 0644 {*}[glob ${worksrcpath}/man/*.1] ${destroot}${prefix}/share/man/man1/
+
+        xinstall -d ${destroot}${prefix}/share/man/man5
+        xinstall -m 0644 {*}[glob ${worksrcpath}/man/*.5] ${destroot}${prefix}/share/man/man5/
+
+        xinstall -d ${destroot}${prefix}/share/${name}
+        xinstall -m 0644 -W ${worksrcpath} README.rst LICENSE ${destroot}${prefix}/share/${name}
+    }
+} else {
+    depends_lib-append \
+                    port:beets
+
+    post-destroot {
+        foreach f {__init__.py __init__.pyc __init__.pyo __pycache__} {
+            set fp "${destroot}${python.pkgd}/beetsplug/$f"
+            if {[file exists ${fp}]} {
+                file delete -force ${fp}
+            }
+        }
+    }
+}
+
+subport beets-alternatives {
+    name            beets-alternatives
+    version         0.10.2.post1
+    revision        0
+
+    license         MIT
+
+    description     beets plugin to manage multiple files
+    long_description ${description}
```

Comment:
the indentation for `long_description` isn't right in every subport; it doesn't follow the "multiple of four spaces"